composer.json

```diff
@@ -4,15 +4,16 @@
     "license": "MIT",
     "require": {
         "nikic/fast-route": "0.7.x",
-        "twig/twig": "1.23.x",
-        "pimple/pimple": "3.0.x",
-        "psr/log": "1.0.0",
-        "monolog/monolog": "1.17.x",
-        "symfony/event-dispatcher": "3.0.x"
+        "twig/twig": "3.3.x",
+        "pimple/pimple": "3.3.x",
+        "psr/log": "1.1.*",
+        "monolog/monolog": "2.2.x",
+        "symfony/event-dispatcher": "5.2.x",
+        "php": ">=7.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7 || ^6.4 || ^7.4",
-        "friendsofphp/php-cs-fixer": "^2.7 !=2.16.1"
+        "phpunit/phpunit": "8.*",
```

Comment:
7.3 doesn't support to 9.* yet

Reply:
Defining both would be better ? e.g. `"^8.0 || ^9.0"`
Then the composer will automatically select the suitable one.

Reply:
9.x works on 7.3, so it'll never choose 8.x